core/server/data/migrations/versions/4.42/2022-03-21-17-17-add.js

```diff
@@ -0,0 +1,20 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('newsletters', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: false},
+    description: {type: 'string', maxlength: 2000, nullable: true},
+    sender_name: {type: 'string', maxlength: 191, nullable: false},
+    sender_email: {type: 'string', maxlength: 191, nullable: false, validations: {isEmail: true}},
+    sender_reply_to: {type: 'string', maxlength: 191, nullable: false, validations: {isEmail: true}},
+    default: {type: 'bool', nullable: false, defaultTo: false},
```

Comment:
Do we have an existing pattern of setting a default bool on a single row vs adding a setting?

Reply:
This field is to be removed ref https://github.com/TryGhost/Team/issues/1474